DTAConfig/Settings/FileSourceDestinationInfo.cs

```diff
@@ -151,6 +134,35 @@ public void Revert()
                         $"Invalid {nameof(FileOperationOptions)} value of {FileOperationOptions}");
             }
         }
+
+        [DllImport("Kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
```

Comment:
Note:

> Also, even if #if NETFRAMEWORK is written, I suggest you make use of SupportedOSPlatform and RuntimeInformation.IsOSPlatform(OSPlatform.Windows) before invoking Win32 APIs since people could also running a .net framework program with wine or mono in linux.

`SupportedOSPlatform` is still missing. You can also find `DefaultDllImportSearchPaths` attributes in the client's source codes.